transcom/mymove
# Mmayo/mb 16588/send email when task order is made available to prime

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16588)

## Summary

This PR changes the workflow of the TOO approving and sending a move to the prime to now email the customer with information about what they can expect moving forward with their interactions with HomeSafe.

Note: It is possible for the TOO to make the move available to the prime and then go back later and approve another shipment. In this case, the email will only be sent on the initial approval and not any subsequent shipment approvals.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

To my knowledge there is no way to send emails from our ephemeral deployments. So the email testing will need to be done on your local machine. You will need to start your local server with the following env variables to utilize our email backend: 
`EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1  aws-vault exec transcom-gov-dev -- make server_run` 

### How to test
1. You need to have a move for a service member that has your personal email. You can do this in multiple ways: 
  * You can create the move from scratch in the customer app and use your personal email. Make one with two shipments that would be helpful later.
  * You can make a fresh dev db and then use this sql query to set all of the emails to your email: `UPDATE service_members SET personal_email = '<your@email.address>';`
2. Login as a TOO to the office app and find the move you created. If you just changed all of the email addresses, then find a New Move: 
![image](https://github.com/transcom/mymove/assets/110134470/e43aebb8-ae70-4a08-8b53-6b8443b2fc71)
3. Approve a shipment: 
![image](https://github.com/transcom/mymove/assets/110134470/0fa1a637-f248-4faa-93cb-d4a1ac1695bb)
4. You should see an email in your inbox: 
![image](https://github.com/transcom/mymove/assets/110134470/75e85a39-04e0-433b-ad5d-356675401b0e)
5. When you approve a second shipment on the same move, you should not see an email.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
